packages/admin/cms-admin/src/generator/future/generateForm/generateFormLayout.ts

```diff
@@ -0,0 +1,53 @@
+import { FormConfig, FormLayoutConfig } from "../generator";
+import { Imports } from "../utils/generateImportsCode";
+
+export function generateFormLayout(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    config: FormLayoutConfig<any>,
+    fieldsCode: string,
```

Comment:
I would recurse instead of passing this as argument. That would give more flexibility for other layouts that might be nested

Reply:
to do this in a clean way this would need some refactoring. I'll do this in a follow-up pull-request.